examples/README.md

```diff
@@ -0,0 +1,14 @@
+# Azure product naming and what example to use:
+
+Today (February 2025) the production, general availability Confidential Containers solution in Azure is Confidential ACI (Azure Container Instances). It is available with two sorts of orchestration, direct ACI (create container groups one at a time and manage them youself) and with AKS using "virtual nodes on Azure Container Instances".
```

Comment:
*generally available Confidential Containers solution

*manage them yourself -- typo on yourself

Should virtual nodes be capitalized as the name/title?

Reply:
oops

Reply:
Done. Now say GA and explain that is "general availability" which seems to be the actual term used by azure.